core/feconf.py

```diff
@@ -40,6 +40,18 @@
 POST_COMMIT_STATUS_PUBLIC = 'public'
 POST_COMMIT_STATUS_PRIVATE = 'private'
 
+
+class ValidCmdDict(TypedDict):
```

Comment:
Why are we defining these in `fconf.py`? maybe `change_domain.py` file?

Reply:
We are defining it in `feconf.py` because this file contains information about some `COMMANDS` and their constants too.

Also after defining it In `feconf.py`, we can use this (`ValidCmdDict`) as a constant type in other files and we can add extra fields if we need by inheriting from `ValidCmdDict`,
like: https://github.com/oppia/oppia/blob/d415824662425e84f4a01766d23b2cf5e9fefbce/core/domain/config_domain.py#L174
https://github.com/oppia/oppia/blob/d415824662425e84f4a01766d23b2cf5e9fefbce/core/domain/exp_domain.py#L279

If we define this `ValidCmdDict` in change_domain.py and assign it to `allowed_commands`, then allowed_command is always considered as a type of ValidCmdDict. So, in future if we need to change the type of allowed_commands for a particular file then most probably we get conflict between the types and we have to use ignore stmt there.